src/material-experimental/mdc-list/list.ts

```diff
@@ -62,6 +64,15 @@ export class MatListItem extends MatListItemBase {
   @ViewChild('unscopedContent') _unscopedContent: ElementRef<HTMLSpanElement>;
   @ViewChild('text') _itemText: ElementRef<HTMLElement>;
 
+  @Input()
+  get activated() {
+    return this._activated;
+  }
+  set activated(activated) {
+    this._activated = activated;
+  }
+  _activated = false;
```

Comment:
Can all of this just be a property? or do you want to add coercion?

Reply:
We typically do coercion for boolean properties so that people can do e.g. `<mat-list-item activated>`

Reply:
Yeah, but currently this getter/setter seems unnecessarily right? could just be a property or do I miss something